meteor/client/ui/Status/package-status/PackageContainerStatus.tsx

```diff
@@ -27,13 +26,28 @@ export const PackageContainerStatus: React.FC<IPackageContainerStatusProps> = fu
 		)
 	}
 
+	const statusMessages: string[] = []
```

Comment:
I guess this should be `offlineReasonMessages` or something? Because L40 directly assumes `.length > 0` means "offline"?